portal-test/src/com/liferay/portal/kernel/test/util/TestPropsUtil.java

```diff
@@ -49,8 +50,8 @@ private TestPropsUtil() {
 			ReflectionUtil.throwException(ioException);
 		}
 
-		try (InputStream inputStream = TestPropsUtil.class.getResourceAsStream(
-				"/test-portal-impl-ext.properties")) {
+		try (InputStream inputStream = PortalClassLoaderUtil.getClassLoader()
+			.getResourceAsStream("/test-portal-impl-ext.properties")) {
```

Comment:
@DaneszLR Please do not chain the method. You should do 
ClassLoader classloader = PortalClassLoaderUtil.getClassLoader();
classloader..getResourceAsStream(